src/sys/signal.rs

```diff
@@ -9,11 +9,13 @@ use std::mem;
 #[cfg(any(target_os = "dragonfly", target_os = "freebsd"))]
 use std::os::unix::io::RawFd;
 use std::ptr;
+use std::ops::{BitOr, BitOrAssign};
 
 // Currently there is only one definition of c_int in libc, as well as only one
 // type for signal constants.
 // We would prefer to use the libc::c_int alias in the repr attribute. Unfortunately
 // this is not (yet) possible.
+
```

Comment:
Please remove this line.

Also, `Signal` needs doc comments now that there's this new behavior where you can combine `Signal`s into `SigSet`s.